### ISSUE TITLE
airodump-ng interpretation of wep/wpa

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#90]
**Reported by darkAudax on 4 Feb 2007 23:12 UTC**

Attached are two beacon packets from the same access point.  You will notice that the mac addresses are only one digit apart.  I have normally seen this on wireless access points that support both A and G(bg).

Here is the info from the airodump-ng text file:
```
00:A0:F8:CE:34:C0, 2007-02-03 10:09:46, 2007-02-03 10:43:50, 11,  54, WEP , WEP,   , 177,     4528,        1,   0.  0.  0.  0,   1,  , 
00:A0:F8:CE:34:C1, 2007-02-03 10:09:47, 2007-02-03 10:43:50, 11,  54, WPA , TKIP, PSK, 178,     4483,      270,   0.  0.  0.  0,   1, O, 
```

You notice that one says wep and the other says wpa.  I believe both are wpa based on looking at the beacon contents.  The only difference is on the tag number 221 which is vendor specific.  One says TKIP and the other says not interpreted.

So I think something is confusing airodump-ng when it determines wep/wpa.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 4 Feb 2007 23:12 UTC**

Easier reading.

**Comment by aircrack-ng** [comment]
**Comment by hirte on 4 Feb 2007 23:12 UTC**

(In #148) Fixed wpa detection with non standard vendor specific flag (Closes #90).

**Comment by aircrack-ng** [comment]
**Modified by misterx on 4 Feb 2007 23:12 UTC**
